makeservices/easywp

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# Creates symlink to user's webroot. If the symlink exists and files
+# are present in webroot, they are backed up in the user's home directory.
+# A user's database is created or, if it already exists, the password 
+# is reset. WordPress is installed and the wp-config file is filled
+# out with the requisite information. Further configuration can be 
+# done by the user from a browser.  
+#
+
+set -euo pipefail
```

Comment:
I still think this script warrants the `-x`, **especially** the part where you back up the user's web dir. If you *really* want to make the script less noisy, I suggest you try a workaround. For example, you can do `set +x` inside a command group:
```bash
set -x
echo hello
(
    set +x
    echo world
)
```
There are alternatives; e.g. you could write a logging helper function and prefix your commands with it.

Reply:
I _still_ think this would be an improvement, but I won't make you do it. At lleast now it displays something for most of the stateful actions it takes.